### PR TITLE
Abstract classes, constants and related functions.

### PR DESCRIPTION
**Pull request by changeling**
I've separated classes, constants and related functions into shared modules for convenience, and to present the primary modules/scripts in a more digestible form.

### CODE REVIEW

**Comment by Linekio** [comment]
Hi,
Thanks for this PR.

I think it would be nice if classes.py was separated into several files, maybe:
- session.py (Session)
- tree.py (Note, Source, Fact, Memorie, Name, Ordinance, Indi, Fam, Tree)
- gedcom.py (Gedcom)
- gui.py (fstogedcom classes)

**Comment by changeling** [comment]
Glad to do it. Take a look at these changes.

There are still some things to consider and probably some to clean up, maybe imports that could be streamlined. For one, I've moved `constants.py` and `translation.py` into the `classes` directory, so strictly speaking, it's not just classes. Maybe that's ok. Thoughts?

How does this look?

File: getmyancestors/classes/gedcom.py [hunk 1]
```diff
@@ -0,0 +1,315 @@
+# mergemyancestors classes
+from getmyancestors.classes.tree import Indi, Fact, Fam, Memorie, Name, Note, Source
+from getmyancestors.classes.constants import FACT_TYPES
+
```
Comment (Linekio) [inline getmyancestors/classes/gedcom.py:4]: A newline is missing here

**Review by Linekio — CHANGES_REQUESTED** [review]

File: getmyancestors/classes/gedcom.py (hunk 1 above)
Comment (changeling) [reply getmyancestors/classes/gedcom.py:14]: Fixed. Also found that class Ordinance and constant ORDINANCES weren't imported in that module.

**Comment by changeling** [comment]
Just checking in.

**Pull request merged**